Source/WebKit/WPEPlatform/wpe/WPEView.cpp

```diff
@@ -643,6 +643,8 @@ gboolean wpe_view_unfullscreen(WPEView* view)
  * wpe_view_render_buffer:
  * @view: a #WPEView
  * @buffer: a #WPEBuffer to render
+ * @damage_rects: (nullable) (array length=n_damage_rects): damage rectangles
```

Comment:
Do the rectangles specify physical buffer coordinates and sizes, or logical ones taking into account the scaling factor? This could use clarification.